testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -98,9 +102,6 @@ type FeeSettings {
   certificateVoteCast: Value!
 }
 
-"""Non-opaque cursor that can be used for offset-based pagination"""
-scalar IndexCursor
-
```

Comment:
Interesting why these scalar types needed to be dropped, but other types remain.

Reply:
Pagination is a feature in `async-graphql`, but not in juniper (where @enzoc4 has to implement himself) iirc.